x-pack/plugin/deprecation/qa/rest/src/javaRestTest/java/org/elasticsearch/xpack/deprecation/DeprecationHttpIT.java

```diff
@@ -629,6 +629,81 @@ public void testCompatibleMessagesCanBeIndexed() throws Exception {
         }
     }
 
+    /**
+     * Check that deprecation messages can be recorded to an index
+     */
+    public void testDeprecationIndexingCacheReset() throws Exception {
+        try {
+            configureWriteDeprecationLogsToIndex(true);
+
+            final Request getRequest = createTestRequest("GET");
+            assertOK(client().performRequest(getRequest));
+
+            client().performRequest(new Request("DELETE", "/_logging/deprecation_cache"));
```

Comment:
Do you think we might ever want to make it possible to clear other caches via the REST API? In which case we could use:

```suggestion
            client().performRequest(new Request("DELETE", "/_cache/deprecation_log"));
```

I'm just thinking off the top of my head, no need to take seriously.

Reply:
I don't think we need to centralize cache access, the caches we have are very independent, this cache is nothing like the query cache, for example.